### DISCUSSION TITLE
how to overriding incoming check

### DISCUSSION THREAD

**Discussion by SadSock** [oracle/opengrok#4481] [Q&A]
![图片](https://github.com/oracle/opengrok/assets/7643216/3fece644-2001-40e7-a6e0-cae80cb85a97)

I use opengrok in official docker image.

thanks

**Comment by vladak** [comment]
This message is part of the normal processing in the Docker image when the mirroring is enabled. If there is a new project added, the incoming check done within the mirroring processing would skip such project because there are no incoming changes. In order to index it, the check is overridden.

Generally speaking, it is not currently possible to configure incoming check for the mirroring processing within the Docker image.
